### ISSUE TITLE
Add details about default value / required constraint to unsupported field comments

### ISSUE THREAD

**Issue by mhwelander** [prisma/prisma#3279]
## Problem
If an unsupported field is required in the underlying DB, you cannot use Prisma Client to write to that table.

## Suggested solution
Add extra info to unsupported field comment, for example:

```
// This type is currently not supported
// This field is required and the default value is 37.774929 -122.419415
// mycol                   geometric?
```

**Comment by janpio** [comment]
What other information than the default value could we put in there?

**Comment by pantharshit00** [comment]
I also not sure how this would be possible if our internal implementation also doesn't handle that type

**Comment by janpio** [comment]
Triage: We can not print the value without supporting the type unfortunately.

**Comment by do4gr** [comment]
This is now implemented as of 2.12
